rust/src/lib.rs

```diff
@@ -1674,3 +1674,16 @@ fn get_index_version(full_version: &str, index: usize) -> Result<String, Error>
         .ok_or(anyhow!(format!("Wrong version: {}", full_version)))?
         .to_string())
 }
+
+// ----------------------------------------------------------
+// Exported functions
+// ----------------------------------------------------------
+
+// this just an example how to expose function for external usage
```

Comment:
@bonigarcia can you please help to implement this function in rust? This function should be equivalent of `selenium-manager.exe --browser chrome --browser-version 130 --browser-path /some/path --proxy some_value`.

Reply:
This is a major change, so there should be a project consensus on this feature first. Personally, I am not sure about it. Are the other bindings (in addition to .Net) following this approach?